packages/course-serializer/src/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const MAX_CATEGORY_NAME_LENGTH = 50;
+export const MAX_WEIGHT = 999_999;
+export const MAX_CREDITS = 5;
+export const MAX_GRADES_PER_CATEGORY = 40;
+export const MINIMUM_CUT = 10;
+export const MAXIMUM_CUT = 10_000;
```

Comment:
Nit:
`MIN_CUTOFF` and `MAX_CUTOFF` would match the rest of the constants

Reply:
Done!